applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -4,6 +4,7 @@
 import KratosMultiphysics.RomApplication as KratosROM
 from KratosMultiphysics.RomApplication import new_python_solvers_wrapper_rom
 from KratosMultiphysics.RomApplication.hrom_training_utility import HRomTrainingUtility
+from KratosMultiphysics.RomApplication.calculate_rom_basis_output_process import CalculateRomBasisOutputProcess
```

Comment:
Why is this required? The process must be imported by the factory when instantiating the content of the `output_process_list`?

Reply:
I noticed the ROM simulation runs well right after the training. But when launching it separately (for changing the number of modes to use for example) Kratos complained that the “calculate_rom_output_process” didn’t belong to the RomApplication. Importing it manually here solved the problem for me..

Reply:
OK. My bet is that if the process is not added in the process list it is not imported, then it cannot do the check below.